tidb-cloud/get-started-with-cli.md

```diff
@@ -0,0 +1,121 @@
+---
+title: TiDB Cloud CLI Overview
+Summary: Learn how to manage TiDB Cloud resources through the TiDB Cloud CLI.
+---
+
+# TiDB Cloud CLI Overview
+
+TiDB Cloud provides a command-line interface (CLI) [`ticloud`](https://github.com/tidbcloud/tidbcloud-cli) to bring deploy cluster requests, import data, and other TiDB Cloud concepts to your fingertips. To interact with TiDB Cloud, you can use the `ticloud` to do the following:
+
+- Create, delete and list your clusters.
+- Import data from S3 or local files to your clusters.
+- ...and more.
```

Comment:
```suggestion
```

Reply:
reason: we can treat the above list as an example to show the most common use cases.